src/Pages/category/CategoryContent/CategoryCarousel/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Carousel } from 'react-responsive-carousel';
+import styles from 'react-responsive-carousel/lib/styles/carousel.min.css';
+import styled from 'styled-components';
```

Comment:
```suggestion
import React from 'react';
import styled from 'styled-components';
import { Carousel } from 'react-responsive-carousel';
import 'react-responsive-carousel/lib/styles/carousel.min.css';
```
- import 순서 맞춰주세요.
- css는 그냥 import 하시면 됩니다.
- 라이브러리에서 import 하는 css 들은 index.js에 모여 있으면 좋을 것 같습니다.

Reply:
수정했습니다!